### ISSUE TITLE
Support monitoring local k8s cluster via Prometheus chart

### ISSUE THREAD

**Issue by vishh** [helm/charts#229]
Existing prometheus charts do not automatically start monitoring the local k8s cluster. Given that a lot of people use prometheus to monitor k8s itself, it will be useful to provide an easy means to configure the chart to start monitoring the local cluster.

Here is a CoreOS guide that sets up prometheus to monitor a local cluster - https://coreos.com/blog/monitoring-kubernetes-with-prometheus.html

I'd argue that given the current state of the k8s project, it might be useful to make the default behavior that of monitoring the local cluster.

**Comment by lachie83** [comment]
cc @mgoodness

**Comment by lachie83** [comment]
Could probably use the following taken from https://coreos.com/blog/the-prometheus-operator.html
- https://coreos.com/operators/prometheus/latest/exporters.yaml
- https://coreos.com/operators/prometheus/latest/prometheus-k8s-cm.yaml

**Comment by viglesiasce** [comment]
This is because GKE uses self signed certs. You need to set the prometheus config to not verify K8s certs in these two lines:

https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L269
https://github.com/kubernetes/charts/blob/master/stable/prometheus/values.yaml#L298

With those bits flipped the cluster will in fact be monitored as you expect.

**Comment by mgoodness** [comment]
@lachie83 I agree that kube-state-metrics would be a good addition to the chart. I'll work on it.

Mixed emotions about node-exporter. It's certainly valuable to have node metrics out of the box via DaemonSet, but I ran into issues deploying that way. There's a comment from @brian-brazil in [this issue](https://github.com/prometheus/node_exporter/issues/66) that rang true for me:

> your problem is that you're trying to run it inside a system designed to prevent the node exporter getting to things like disk stats

I ended up installing directly on the nodes. Tears were shed.
